src/utils/url.ts

```diff
@@ -0,0 +1,7 @@
+const prefixUrlWithHttp = (url: string) => {
+  if (!url.startsWith('http')) return url;
```

Comment:
```suggestion
  if (url.startsWith('http')) return url;
```

Shouldn't it be 
if url.startsWith('http') -> return url;
else -> return `http://${url}`;

Reply:
Ooh yes 🤦